docs/sdk/push/android.md

```diff
@@ -93,20 +96,27 @@ The App Center SDK is designed with a modular approach – a developer only need
 
 In order to use App Center, you need to opt in to the module(s) that you want to use, meaning by default no modules are started and you will have to explicitly call each of them when starting the SDK.
 
-Add `Push.class` to your `AppCenter.start()` method to start App Center Push.
+Add the `Push` module to your `AppCenter.start()` method to start App Center Push.
```

Comment:
You add the module to the project, but this call initializes the push class, right?  Is this really as clear as it should be?

Reply:
Could also say "class" here - I wanted to make sure to not say "Push.class" because it's language dependent. Will change this to "`Push` class" or even provide a sample for both languages.